runtime/testy/runtime.py

```diff
@@ -114,7 +114,7 @@ def nonTestModePrint(*args):
           break
       stateQueue.put([SM_COMMANDS.RESET, []])
       terminate_process(PROCESS_NAMES.STUDENT_CODE)
-      stateQueue.put([SM_COMMANDS.SET_VAL, [runtime_pb2.RuntimeData.STUDENT_STOPPED, ["studentCodeState"]]])
+      stateQueue.put([SM_COMMANDS.SET_VAL, [runtime_pb2.RuntimeData.STUDENT_STOPPED, ["studentCodeState"], False]])
```

Comment:
`set=False` will be more readable i feel. 
Personally i'm a fan of always naming keyword args in function calls, because the fact that theyre optional can be confusing on first read.